index.js

```diff
@@ -0,0 +1,28 @@
+const jwt = require("jsonwebtoken");
+const express = require('express');
+const app = express();
+const port = 8080;
+
+const METABASE_EMBEDDING_SECRET = process.env.METABASE_EMBEDDING_SECRET;
+const METABASE_SITE_URL = process.env.METABASE_SITE_URL || 'http://localhost:3000';
+const METABASE_EMBED_DASHBOARD_ID= parseInt(process.env.METABASE_EMBED_DASHBOARD_ID);
+
+app.get('/', (req, res) => {
+  var payload = {
+    resource: { dashboard: METABASE_EMBED_DASHBOARD_ID },
+    params: {},
+    exp: Math.round(Date.now() / 1000) + (10 * 60) // 10 minute expiration
+  };
+  var token = jwt.sign(payload, METABASE_EMBEDDING_SECRET);
+
+  var iframeUrl = new URL("/embed/dashboard/" + token, METABASE_SITE_URL);
+  iframeUrl.hash = "bordered=true&titled=true";
+  res.send(
+    `<iframe src="${iframeUrl}" frameborder="0" width="1280" height="1000" allowtransparency></iframe>`
+  );
```

Comment:
I don't know the embeddings domain well enough but this seems ok. Easy to understand - which I suppose is what is required.

However, I'd change `var`s inside the callback to `const`s too.

Reply:
Yes, easy to understand is what we are aiming for!

Reply:
I will replace var for const.